### DISCUSSION TITLE
2009/09/06/%E7%AD%BE%E5%90%8D

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#275] [Announcements]
# 2009/09/06/%E7%AD%BE%E5%90%8D

最近一想起签名就头疼，因为我写字实在太难看了。一般人就算写字难看，自己的名字总还是写得不错的，毕竟这几个字练得最多。可是我却连名字也写不好看。其实我上学的时候也曾经正儿八经练过毛笔和钢笔书法的，可就是没什么长进。毕业之后，几乎都用不到写字了，就更不可能有进步了。

https://blog.qizhen.xyz/2009/09/06/%E7%AD%BE%E5%90%8D

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Chocolate
2009年10月14日 12:52 编辑
那就放轻松就写啊！~呵呵

回复
Qizhen
2009年10月14日 09:50 编辑
也许可以搞个抽奖了，看谁恰好拿到了图片上这本书。

回复
帆
2009年10月13日 12:27 编辑
其实啊，我收到的签名版上的字比这个要好。

回复
Qizhen
2009年10月13日 11:24 编辑
感谢你的支持:)

回复
增光
2009年10月10日 11:05 编辑
上个星期买了您的书，是在中关村图书大厦买的，呵呵。之前也看过几本其它labview相关的书，觉得还是您的看起来比较好。不过个人愚见，觉得您的书不太适合入门阅读，里面的经验之谈和高级处理技巧，还是需要在平时练习中去慢慢消化的，没有一定的基础还是很难接受的。希望以后能和您有更多的交流~~~ 增光

回复
光欣
2009年09月8日 09:45 编辑
只要你认真写了就好，欢迎加我MSN

回复
晖英
2009年09月7日 11:05 编辑
呵呵，有自己的特色，也不错啊。

回复
奕瑋
2009年09月7日 10:08 编辑
看你的字，比较安静，比那些行草、潦草之类的浮躁签名舒服的多。

回复
l
2009年09月7日 00:12 编辑
以后NI开研讨会可以考虑奖品送阮大侠亲笔签名的LabVIEW秘籍,一定会很受欢迎.

回复
